examples/website/data-filter/app.js

```diff
@@ -22,9 +22,12 @@ const INITIAL_VIEW_STATE = {
   bearing: 0
 };
 
-const MS_PER_DAY = 8.64e7; // milliseconds in a day
+const MS_PER_DAY = 8.64e7;
 
-const dataFilter = new DataFilterExtension({filterSize: 1});
+const dataFilter = new DataFilterExtension({
+  filterSize: 1,
+  fp64: false // enable for higher precision
```

Comment:
Should this be `true` now that the filtering values are so much larger? (not dividing by `MS_PER_DAY` anymore)

Reply:
The example itself doesn't show any visible precision issue because the filter is moving 30 days per frame, and the error by using f32 is only 1-2 minutes.